### PR TITLE
Add new method for writing to data_feed for Byteball light

### PR DESCRIPTION
**Pull request by SanderBuK**
We don't know if this should be merged, as it would override the AAInteractions, which are to be used in the next sprint

### CODE REVIEW

**Review by thomasdal — APPROVED** [review]

File: src/utils/writeToDataFeed.ts [hunk 1]
```diff
@@ -11,7 +10,7 @@ export function writeToDataFeed(dataFeed: unknown) {
     state.wallet.issueOrSelectAddressByIndex(0, 0, (botWallet) => {
         const params = {
             paying_addresses: [botWallet],
-            outputs: [{ address: botWallet, amount: 0 }],
+            outputs: [{ address: botWallet, amount: 10000 }],
```
Comment (nimrossum) [inline src/utils/writeToDataFeed.ts]: What is the reason for this change?

Comment (SanderBuK) [reply src/utils/writeToDataFeed.ts]: Originally it was a required amount to send a transaction, but since the new method doesn't use it, I've deleted it completely.

**Review by WizOfRob — APPROVED** [review]
Doesn't break DoD, nice.
